ci-operator/step-registry/ipi/conf/capability/additonalcaps/ipi-conf-capability-additonalcaps-commands.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ "${BASELINE_CAPABILITY_SET}" == "" ]]; then
+  echo "This step requires BASELINE_CAPABILITY_SET to be set!"
+  exit 1
+fi
+
+echo "RELEASE_IMAGE_LATEST: ${RELEASE_IMAGE_LATEST}"
+echo "RELEASE_IMAGE_LATEST_FROM_BUILD_FARM: ${RELEASE_IMAGE_LATEST_FROM_BUILD_FARM}"
+export HOME="${HOME:-/tmp/home}"
+export XDG_RUNTIME_DIR="${HOME}/run"
+export REGISTRY_AUTH_PREFERENCE=podman # TODO: remove later, used for migrating oc from docker to podman
+mkdir -p "${XDG_RUNTIME_DIR}"
+# After cluster is set up, ci-operator make KUBECONFIG pointing to the installed cluster,
+# to make "oc registry login" interact with the build farm, set KUBECONFIG to empty,
+# so that the credentials of the build farm registry can be saved in docker client config file.
+# A direct connection is required while communicating with build-farm, instead of through proxy
+KUBECONFIG="" oc --loglevel=8 registry login
+ocp_version=$(oc adm release info ${RELEASE_IMAGE_LATEST_FROM_BUILD_FARM} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+echo "OCP Version: $ocp_version"
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+
+v411="baremetal marketplace openshift-samples"
+v412=" ${v411} Console Insights Storage CSISnapshot"
+v413=" ${v412} NodeTuning"
+# shellcheck disable=SC2034
+v414=" ${v413} MachineAPI Build DeploymentConfig ImageRegistry"
+
+# shellcheck disable=SC2034
+v_current_version="v${ocp_major_version}${ocp_minor_version}"
+if (( ocp_minor_version == 11 )); then
+    v_pre_version=""
+else
+    v_pre_version="v${ocp_major_version}$((ocp_minor_version-1))"
+fi
+
+new_capabilities="${!v_current_version}"
+for cap in ${!v_pre_version}; do
+    new_capabilities=${new_capabilities/$cap}
+done
+
+if [[ -z ${new_capabilities} ]]; then
+    echo "No new added capabilities on OCP ${ocp_version}, exit..."
+    exit 1
```

Comment:
here so we only test the incremental capacities? How about selecting a random capacity to run the testing? But not have to care about the failure caused by no incremental capacities.

Reply:
[current test case](https://gitlab.cee.redhat.com/aosqe/v4-installer-autotest/-/blob/master/cases/TestCases/OCP-49192/Install_cluster_baselineCapabilitySet_None_additionalEnabledCapabilities.py) is to only test the incremental capabilities, so migrate the same to prow.

I remembered that we discussed this before and thought it should be enough to only test incremental capabilities on target payload, other than testing duplicated scenarios on different payload(e.g, enable capability baremetal on 4.11/4.12/4.13/4.14...). cc: @gpei @shellyyang1989

Reply:
Yeah, indeed. But for now, we are using a random capacity to run our testing, and it becomes easier for a full coverage now, and we can run the same testing on different versions, we also can get more confidence on that coverage.

Reply:
@jinyunma I went through our discussion again, seems the more considerate testing way here is randomly disable one of the new_capabilities for each install?

Reply:
another thing is if we decide to randomly selecting the capability for this job `gcp-ipi-baselinecaps-none-additionalcaps-f28` , maybe we need to increase the frequency higher, to make sure all items could be tested before 4.14 GA, suggest using `f7` for it

Reply:
correct, it is indeed disabling each new_capabilities instead of only enabling it in v4-installer-autotest case script. Updated same here. I changed f28 to f14, let me updated to f7, but I think we can change back to f14 after GA.

Reply:
"disable one" v.s "enable one", both is okay for me, because we always cover baseline=None, "enable one" maybe also become easy to debug.

The another advantage for "enable one", maybe we more easier for coding, on your call.